tests/test_coverage.py

```diff
@@ -205,3 +206,16 @@ def test_logging_partially_documented_file(caplog, expected, verbose, ignore_nam
         )
 
     assert caplog.messages == expected
+
+
+def test_skip_private():
```

Comment:
@HunterMcGushion this test demonstrates what the `skip_private` argument is supposed to do, but I got unexpected behavior when there was no module docstring. Have you got any idea on why that's the case?

Reply:
Ahh... Many of the existing tests rely on the contents of `sample_files`, so it looks like adding a new sample file messed up those tests. Can you try moving your new sample file into the "sample_files" directory, rather than "subdir_a"? If that doesn't work, it may need to be outside "sample_files" completely, which is a shame, but I think it's worth it to get your new feature added.

Reply:
Reorganized in 3c0779d, hope that fixes it

Reply:
It would be good to set up something like tox to make tests more reproducible

Reply:
Hahaha no you're right, I should definitely get familiar with Tox. Glad you got this worked out!